### PR TITLE
Holocene: e2e test for Holocene's configurable eip-1559 params

### PR DESCRIPTION
**Pull request by roberto-bayardo**
**Description**

Provides an E2E test that exercises the updating of the EIP-1559 parameters through SystemConfig contract, ensuring the base fee gets computed as expected both before and after the update.

### CODE REVIEW

File: op-e2e/system/fees/eip1559params_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	require.Equal(t, expectedElasticity, elasticity)
+
+	_, err = geth.WaitForL1OriginOnL2(sys.RollupConfig, receipt.BlockNumber.Uint64(), l2Seq, txTimeoutDuration)
+	require.NoError(t, err, "waiting for L2 block to include the sysconfig update")
+
+	h, err := l2Seq.HeaderByNumber(context.Background(), nil)
+	require.NoError(t, err)
+
+	// confirm the extraData is being set as expected
+	eip1559.DecodeHoloceneExtraData(h.Extra)
```
Comment (tynes) [inline op-e2e/system/fees/eip1559params_test.go]: This skips assigning the return values to make sure it doesn't panic?

**Review by tynes — APPROVED** [review]
This looks good to me!

File: op-e2e/system/fees/eip1559params_test.go (hunk 1 above)
Comment (sebastianst) [reply op-e2e/system/fees/eip1559params_test.go]: You could then wrap it in a `require.NoPanic`.

File: op-e2e/system/fees/eip1559params_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	require.Equal(t, eip1559.EncodeHoloceneExtraData(uint64(expectedDenom), uint64(expectedElasticity)), h.Extra)
+
+	// confirm the next base fee will be as expected with the new 1559 parameters
+	delta := ((15_000_000 - int64(h.GasUsed)) * h.BaseFee.Int64() / 15_000_000 / int64(expectedDenom))
```
Comment (sebastianst) [inline op-e2e/system/fees/eip1559params_test.go]: would be nicer to read with the 15M be defined as a `const` where you define `expectedElasticity`.

**Review by sebastianst — APPROVED** [review]
LGTM just two nits

File: op-e2e/system/fees/eip1559params_test.go (hunk 2 above)
Comment (roberto-bayardo) [reply op-e2e/system/fees/eip1559params_test.go]: done

File: op-e2e/system/fees/eip1559params_test.go (hunk 1 above)
Comment (roberto-bayardo) [reply op-e2e/system/fees/eip1559params_test.go]: actually had put that in there during debugging to log the output, that method won't panic so it's just dead code. removed.

**Pull request merged**